docs/tutorials/multimodal/index.md

```diff
@@ -113,6 +113,20 @@ In the following, we decomposed the functionalities of AutoMM and prepared step-
 ::::
 
 
+<<<<<<< HEAD
```

Comment:
Are we intended to keep this or it is leftover from conflict resolution?

Reply:
Removed.